### PR TITLE
feat/US34176: safe-list-settings

### PR DESCRIPTION
**Pull request by roshan-bisht-lab49**

### CODE REVIEW

File: src/dex-ui/pages/utils.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+export const isValidUrl = (urlString: string) => {
```
Comment (damiano-melcarne-lab49) [inline src/dex-ui/pages/utils.ts:1]: These functions are generically useful. We may want to move these functions to the top-level shared utils folder `@utils` if there used across multiple components.

File: src/dex-ui/pages/dao/MultiSigDAODashboard/Settings.tsx [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+                    THRESHOLD
+                  </Text>
+                  <Text textStyle="p large medium">{`${threshold} / ${memberCount}`}</Text>
+                </Flex>
+                <Button
+                  type="button"
+                  variant="primary"
+                  padding="10px 10px"
+                  height="40px"
+                  onClick={() => handleChangeThresholdClick()}
```
Comment (damiano-melcarne-lab49) [inline src/dex-ui/pages/dao/MultiSigDAODashboard/Settings.tsx:169]: Can just be `onClick={handleChangeThresholdClick}`

File: src/dex-ui/pages/dao/MultiSigDAODashboard/Settings.tsx [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+          <Flex direction="column" gap={2} marginBottom="0.4rem">
+            <Text textStyle="p medium medium">Members</Text>
+            <Text textStyle="p small regular" marginBottom="0.7rem" color={Color.Neutral._500}>
+              Manage the member preferences, add, remove or rename them.
+            </Text>
+            <Divider />
+          </Flex>
+          <Flex direction="row" alignItems="center" marginBottom="0.4rem" justifyContent="space-between">
+            <Text textStyle="p small regular">{memberCount} Members</Text>
+            <Button type="button" variant="primary" onClick={() => handleAddNewMemberClicked()}>
```
Comment (damiano-melcarne-lab49) [inline src/dex-ui/pages/dao/MultiSigDAODashboard/Settings.tsx:86]: Can be onClick={handleAddNewMemberClicked}

**Review by damiano-melcarne-lab49 — APPROVED** [review]

File: src/dex-ui/pages/dao/MultiSigDAODashboard/Settings.tsx (hunk 2 above)
Comment (roshan-bisht-lab49) [reply src/dex-ui/pages/dao/MultiSigDAODashboard/Settings.tsx]: Done

**Review by damiano-melcarne-lab49 — APPROVED** [review]

**Pull request merged**